Hi, the **Pre_test Static Analysis** pre-checker is failing on this PR due to a flake8 lint error in your change:

```
tests/vxlan/test_vnet_bgp_route_precedence.py:372:50: E231 missing whitespace after ':'
```

`/azp run` won't clear this since it's a real lint error in the code — please fix it (run `python3 -m flake8 <file> --max-line-length=120` locally to confirm clean), then push an update. Thanks!

--- a/tests/vxlan/test_vnet_bgp_route_precedence.py
+++ b/tests/vxlan/test_vnet_bgp_route_precedence.py
@@ -128,10 +128,16 @@ def _ignore_route_sync_errlogs(duthosts, rand_one_dut_hostname, loganalyzer):
             ".*_M_construct null not valid.*",
             ".*construction from null is not valid.*",
             ".*meta_sai_validate_route_entry.*",
-            ".*ERR.* flush_creating_entries: EntityBulker.flush create entries failed.*",
-            ".*ERR.* start: Encountered failure in create operation.*",
-            ".*ERR.* addRoutePost: Failed to create route.*",
-
+            # The test intentionally programs an overlapping BGP/VNET route for
+            # the same prefix, so orchagent's bulk route create hits
+            # SAI_STATUS_ITEM_ALREADY_EXISTS and the rest of the bulk returns
+            # SAI_STATUS_NOT_EXECUTED. Scope each ignore to the specific SAI
+            # status / route context so it cannot mask unrelated route failures.
+            ".*ERR.* flush_creating_entries: EntityBulker.flush create entries failed.*"
+            "status: SAI_STATUS_ITEM_ALREADY_EXISTS.*",
+            ".*ERR.* Encountered failure in create operation, SAI API: SAI_API_ROUTE,"
+            " status: SAI_STATUS_NOT_EXECUTED.*",
+            ".*ERR.* addRoutePost: Failed to create route .* with next hop.*",
         ]
         # Ignore in KVM test
         KVMIgnoreRegex = [

--- a/tests/vxlan/test_vnet_bgp_route_precedence.py
+++ b/tests/vxlan/test_vnet_bgp_route_precedence.py
@@ -128,6 +128,9 @@ def _ignore_route_sync_errlogs(duthosts, rand_one_dut_hostname, loganalyzer):
             ".*_M_construct null not valid.*",
             ".*construction from null is not valid.*",
             ".*meta_sai_validate_route_entry.*",
+            ".*ERR.* flush_creating_entries: EntityBulker.flush create entries failed.*",
+            ".*ERR.* start: Encountered failure in create operation.*",
+            ".*ERR.* addRoutePost: Failed to create route.*",
 
         ]
         # Ignore in KVM test

